app/views/forums/index.html.haml

```diff
@@ -7,7 +7,7 @@
 - @forums.each do |forum|
   %h2= forum
   %p
-    = pluralize(forum.posts.size, "post")
+    = localize_plural(forum.posts, Post)
```

Comment:
So, we have a collection of Posts already passed in; do we need to inject the model explicitly? I imagine 'yes' or 'its difficult' for empty collections, but for others do we have the ability to look at ```.first.class.name```?